src/main/java/info/sjd/dao/ItemDAO.java

```diff
@@ -11,7 +12,12 @@
 public interface ItemDAO extends JpaRepository<Item, Integer> {
     List<Item> getByCode(String code);
 
-//    List<Item> getAllByCart(Cart cart);
+    @Query(value = "SELECT ALL FROM items i " +
```

Comment:
Wrong query. ALL must be changed to parameters list or *

Reply:
bug fixed